packages/create-cli/package.json

```diff
@@ -40,19 +40,19 @@
   },
   "homepage": "https://github.com/checkly/checkly-cli#readme",
   "dependencies": {
+    "@esm2cjs/execa": "6.1.1-cjs.1",
     "@oclif/core": "2.8.11",
     "@oclif/plugin-help": "5.1.20",
     "@oclif/plugin-plugins": "2.3.0",
     "axios": "1.3.1",
     "chalk": "4.1.2",
     "debug": "4.3.4",
-    "execa": "5.1.1",
-    "fullname": "4.0.1",
     "giget": "1.1.2",
     "ora": "5.4.1",
+    "passwd-user": "3.0.0",
```

Comment:
The package readme says it doesn't work on windows. Have you checked that out?

Reply:
@umutuzgur , yes yes, it worked. That `passwd-user` package is one of the multiple alternatives to get the username. I tried the `create-cli` on Windows and Linux and it can get the fullname:
![image](https://github.com/checkly/checkly-cli/assets/5315705/abd859f0-6c90-485f-ae4c-a1b10d06cf74)